code/modules/projectiles/projectile/energy/net_snare.dm

```diff
@@ -1,28 +1,32 @@
-/obj/projectile/energy/net
-	name = "energy netting"
+/obj/projectile/energy/snare
+	name = "energy snare"
 	icon_state = "e_netting"
-	damage = 10
+	damage = 30
 	damage_type = STAMINA
 	hitsound = 'sound/items/weapons/taserhit.ogg'
 	range = 10
 
-/obj/projectile/energy/net/Initialize(mapload)
+/obj/projectile/energy/snare/Initialize(mapload)
 	. = ..()
 	SpinAnimation()
 
-/obj/projectile/energy/net/on_hit(atom/target, blocked = 0, pierce_hit)
+/obj/projectile/energy/snare/on_hit(atom/target, blocked = 0, pierce_hit)
 	var/obj/item/dragnet_beacon/destination_beacon = null
 	var/obj/item/gun/energy/e_gun/dragnet/our_dragnet = fired_from
 	if(our_dragnet && istype(our_dragnet))
 		destination_beacon = our_dragnet.linked_beacon
 
 	if(isliving(target))
-		var/turf/Tloc = get_turf(target)
-		if(!locate(/obj/effect/nettingportal) in Tloc)
-			new /obj/effect/nettingportal(Tloc, destination_beacon)
+		var/mob/living/living_target = target
+		living_target.adjust_staggered_up_to(STAGGERED_SLOWDOWN_LENGTH * 0.5, 5 SECONDS)
+		for(var/turf/trapped_turf in range(1, get_turf(living_target)))
+			if(trapped_turf.density)
```

Comment:
Do you want to place portals on turfs covered by dense objects such as lockers, computers, tables, machines, etc?
If that's fine then checking turf density is sufficient, but otherwise you might need `is_blocked_turf`

Reply:
I think tables and possibly lockers and crates may be legitimate targets given players can climb those. So I will probably say yes.